src/config.js

```diff
@@ -0,0 +1,9 @@
+import { config as favicons } from 'favicons';
+
+const config = {
+  onSuccess() {},
+  iconPath: 'favicon.png',
```

Comment:
Shouldn't this default to something like `public/favicons.png`? I don't think the source image belongs at the project root level.

Reply:
Not really. This project should be fully independent, and not aware of Ember structure. It's working on a passed input and it should look in that input for the icon directly.

Reply:
Ah right, `ember-cli-favicon` should default to that, good point 👍